lambdas/api/put_dmps_uploads/src/__tests__/index.spec.ts

```diff
@@ -0,0 +1,138 @@
+const mockGetExport = jest.fn();
```

Comment:
Is there a way to run these unit tests locally? Whenever I try I get the error "Cannot find module '@aws-sdk/client-cloudformation'".

Reply:
yes, you need to be in the `lambdas/api/put_dmps_uploads` dir and run `npm install` first then `npm run test` that should install all of those dev dependencies.

Reply:
I did that, but still see that error for some reason.

Reply:
Hmm. try going into the `lambda/layers/dmptool/cloudformation` directory and run `npm install`. Then go back to the `lambda/api/put_dmps_uploads` dir and run the tests again. They reference those lambda layers directly when you run locally so I wonder if the error is actually coming from the layer

Reply:
Thanks @briri. That worked